member.go

```diff
@@ -70,6 +70,17 @@ func (c *Client) Member(nick string) (member *Member, err error) {
 	return
 }
 
+func (c *Client) MemberFromToken(token string) (member *Member, err error) {
```

Comment:
For current user, we could use `/members/me` instead.

Is there any strong use case for `MemberFromToken()`? Or can we just do

```go
for _, token := range tokens {
   c, err := NewAuthClient(appKey, token)
   //...
   member, err := c.Member()
   //...
}
```